src/main/java/org/apache/gossip/manager/Shutdownable.java

```diff
@@ -0,0 +1,5 @@
+package org.apache.gossip.manager;
+
+public interface Shutdownable {
```

Comment:
Could we consider Closable? I do not think if we remove the reflection we need this.